packages/next/client/future/image.tsx

```diff
@@ -351,6 +352,8 @@ const ImageElement = ({
         className={className}
         // @ts-ignore - TODO: upgrade to `@types/react@17`
         loading={loading}
+        // https://web.dev/priority-hints/#increase-the-priority-of-the-lcp-image
+        fetchPriority={priority ? "high" : undefined}
```

Comment:
undefined should probably be changed to `rest.fetchPriority` so we don't break anyone who is currently setting this prop to a custom value like `"low"` for example.

Reply:
> undefined should probably be changed to `rest.fetchPriority` so we don't break anyone who is currently setting this prop to a custom value like `"low"` for example.

Done :)